pkg/gateway/sig/v4.go

```diff
@@ -33,6 +33,15 @@ const (
 	v4scopeTerminator        = "aws4_request"
 	v4timeFormat             = "20060102T150405Z"
 	v4shortTimeFormat        = "20060102"
+	maxExpires               = 604800
```

Comment:
This needs to be more explicit. Either by documentation or parameter name
```suggestion
	AmzPresignMaxExpires               = 604800
```